commands/shooter/aimshootermanual.py

```diff
@@ -15,11 +15,14 @@ def __init__(self, shooter: ShooterSubsystem, shooterOffset: Control2D) -> None:
         self.shooter = shooter
         self.distanceAndRotationOffset = shooterOffset
         self.addRequirements([self.shooter])
+        SmartDashboard.putNumber(constants.kShootingHoodAngleKey, 8)
 
     def execute(self) -> None:
-        wheelSpeed = SmartDashboard.getNumber(constants.kShootingWheelSpeedKey, 0)
-        hoodAngle = SmartDashboard.getNumber(constants.kShootingHoodAngleKey, 0)
-        turretPosition = SmartDashboard.getNumber(constants.kShootingTurretAngleKey, 0)
+        # wheelSpeed = SmartDashboard.getNumber(constants.kShootingWheelSpeedKey, 550)
+        # hoodAngle = SmartDashboard.getNumber(constants.kShootingHoodAngleKey, 8)
+        wheelSpeed = 550
```

Comment:
Probably make these smartdash again for weekend testing

Reply:
migrated to dedicated fixed values in #83